canvas/migrations/0015_canvascourse_description.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.0.14 on 2022-10-22 19:24
```

Comment:
This migration file has a conflict because I've added a new one recently. Please do the following in your local machine.
1. Revert this migration (Before pulling from master)
2. Delete the migration file (Before pulling from master)
3. Pull from master (the main one)
4. Generate the migration
5. Run the migration
6. Make sure everything is ok